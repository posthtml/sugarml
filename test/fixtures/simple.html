doctype html
div.bar(id='test' @wow data-foo='bar')
  p#foo hello world!
  .wow
    .nest2
      .nest3
        p more text
  .unnest
  // a comment
  li: a(href='wow') content
  p
    | text on its own line
    span now it's a span!
    | ah, more text on its own line...
