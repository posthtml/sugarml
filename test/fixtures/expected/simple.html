<!DOCTYPE html>
<div class="bar" id="test" @wow="" data-foo="bar">
  <p id="foo">hello world!</p>
  <div class="wow">
    <p>more text</p>
  </div><div class="unnest">
  </div><!-- a comment -->
  <li><a href="wow">content</a></li>
  <p>
    text on its own line
    <span>now it's a span!</span>
    ah, more text on its own line...
</p></div>
